docs/manual/java/releases/Migration13.md

```diff
@@ -0,0 +1,13 @@
+# Lagom 1.3 Migration Guide
+
+This guide explains hot to migrate from Lagom 1.2 to Lagom 1.3
```

Comment:
I also added the migration page for 1.3. The change is small but preferred documenting it. Most users won't be affected because the only non-backwards compatible change affects consuming-only apps (on runtime).